src/03-modality-agnostic-files.md

```diff
@@ -81,29 +85,105 @@ Template:
 ```Text
 participants.tsv
 participants.json
-phenotype/<measurement_tool_name>.tsv
-phenotype/<measurement_tool_name>.json
 ```
 
-Optional: Yes
+The purpose of this RECOMMENDED file is to describe properties of participants
+such as age, sex, handedness etc.
+In case of single session studies this file has one compulsory column
+`participant_id` that consists of `sub-<label>`, followed by a list of optional
+columns describing participants.
+Each participant MUST be described by one and only one row.
+
+Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`,
```

Comment:
This is a bit confusing. [Optional is different from recommended](https://www.ietf.org/rfc/rfc2119.txt). So which one is it?

Reply:
hopefully cleared up in https://github.com/bids-standard/bids-specification/pull/459/files#r417826941